app/Main.hs

```diff
@@ -50,11 +50,16 @@ import qualified Language.Fortran.Parser.Free.Lexer  as Free
 programName :: String
 programName = "fortran-src"
 
+showVersion :: String
+showVersion = "0.16.0"
+
 main :: IO ()
 main = do
   args <- getArgs
   (opts, parsedArgs) <- compileArgs args
   case (parsedArgs, action opts) of
+    (paths, ShowMyVersion) -> do
+      putStrLn $ "fortran-src version: " ++ showVersion
```

Comment:
(We should have a release guide that instructs updating the version string shown here. I'll start a wiki page later.)